torch/ao/quantization/fx/_model_report/model_report_observer.py

```diff
@@ -0,0 +1,84 @@
+import torch
+from torch.ao.quantization.observer import ObserverBase
+
+
+class ModelReportObserver(ObserverBase):
+    r"""This observer is used to record additional information regarding keeping track
+    of S = average_batch_activation_range/epoch_activation_range.
+
+    The purpose of this information is to prepare a report to present to users on whether
+    Dynamic or Static Quantization is more appropriate for their model given the general
+    distributions of their data.
+
+    * :attr:`num_batches_tracked` specifies number of batches passed through the observer
```

Comment:
sorry, let's use this format: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/observer.py#L348 (rednered: https://pytorch.org/docs/stable/generated/torch.quantization.observer.MinMaxObserver.html#torch.quantization.observer.MinMaxObserver), we might want to fix the FakeQuantize one since that does not look nice: https://pytorch.org/docs/stable/generated/torch.quantization.fake_quantize.FakeQuantize.html#torch.quantization.fake_quantize.FakeQuantize

Reply:
sorry, I think we probably need to use both, see https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/conv.py#L671-L731, attr for the attribute in the class and args for the arguments to `__init__` function, in this case we don't have args, so current code looks good